crates/spicepod/src/component/tool.rs

```diff
@@ -30,7 +31,7 @@ pub struct Tool {
     pub description: Option<String>,
 
     #[serde(default, skip_serializing_if = "HashMap::is_empty")]
-    pub params: HashMap<String, String>,
+    pub params: HashMap<String, Value>,
```

Comment:
I think we should just make `env` a sibling to `params` and pass it down to any dependent tools.